stdlib/3/shlex.pyi

```diff
@@ -2,15 +2,18 @@
 
 # Based on http://docs.python.org/3.2/library/shlex.html
 
-from typing import List, Tuple, Any, TextIO, Union, Optional
+from typing import List, Tuple, Any, TextIO, Union, Optional, TypeVar, Iterator
+import sys
+
+_T = TypeVar('_T')
 
 def split(s: str, comments: bool = ...,
           posix: bool = ...) -> List[str]: ...
 
 # Added in 3.3, use (undocumented) pipes.quote in previous versions.
 def quote(s: str) -> str: ...
 
-class shlex:
+class shlex(Iterator[_T]):
```

Comment:
`_T` is unnecessary, this should simply be `Iterator[str]` since this is the only supported type of output.